diff --git a/dist/uni-h5.es.js b/dist/uni-h5.es.js
index 7421bad97d94ad34a3d4d94292a9ee9071430662..19c6071ee4036ceb8d1cfa09030e471c002d2cda 100644
--- a/dist/uni-h5.es.js
+++ b/dist/uni-h5.es.js
@@ -23654,7 +23654,7 @@ function useShowTabBar(emit2) {
   const tabBar2 = useTabBar();
   const showTabBar2 = computed(() => route.meta.isTabBar && tabBar2.shown);
   updateCssVar({
-    "--tab-bar-height": tabBar2.height
+    "--tab-bar-height": tabBar2?.height || 0
   });
   return showTabBar2;
 }
